aiosonic/__init__.py

```diff
@@ -778,7 +778,7 @@ async def request(
         urlparsed = _get_url_parsed(url)
 
         boundary = None
-        headers = deepcopy(headers) if headers else []
+        headers = HttpHeaders(headers) if headers else []
```

Comment:
```suggestion
        headers = HttpHeaders(deepcopy(headers)) if headers else []
```

Better to maintain the deepcopy I think :) so we don't modify provided headers object

Reply:
Oh, I see. it's fixed. thanks!